include/swift/Parse/Tokens.def

```diff
@@ -173,9 +187,19 @@ POUND_KEYWORD(endif)
 POUND_KEYWORD(line)
 POUND_KEYWORD(setline)
 POUND_KEYWORD(sourceLocation)
-POUND_KEYWORD(available)
 POUND_KEYWORD(selector)
 
+// Keywords in the # namespace that are build configurations.
+POUND_CONFIG(available)
+
+// FIXME: It feels like a layering violation to mention the protocol
+// kind and description here, but essentially there is a single
+// delcarative way to specify the shape of object literals, their
```

Comment:
typo "declarative".  

I think this approach is reasonable, no need for the FIXME.